### DISCUSSION TITLE
Видео перестало грузить более 59 сек на PC, андроид работает

### DISCUSSION THREAD

**Discussion by wildarp** [bol-van/zapret#963] [Q&A]
Перестал идти поток видео, точнее идет до 59сек и все виснет, может кто-то сталкивался ?

**Comment by XZVB12** [comment]
Сталкивался, что zapret, что byeDPI, одинаковый результат на разных конфигах. Идей, что может быть причиной, пока нет.
[Тут разгоняли тему про якобы бан от гугла](https://ntc.party/t/%D1%81%D0%B1%D0%BE%D1%80%D0%BA%D0%B0-ytdisbystro-%D0%BD%D0%B0-%D0%BE%D1%81%D0%BD%D0%BE%D0%B2%D0%B5-zapret-%D0%B4%D0%BB%D1%8F-windows-%D0%BE%D0%B1%D1%81%D1%83%D0%B6%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5/13251/18), крайне сомнительное заявление,в браузере без акаунта у меня такой же результат, но смена клиента действительно помогает.
На реддите тоже обсуждали:https://www.reddit.com/r/youtube/comments/1gw6zhb/youtube_videos_stop_loading_after_a_minute/?sort=new
https://www.reddit.com/r/youtube/comments/1epk1fj/help_videos_not_playing_beyond_the_1_min_mark/?sort=new

**Comment by BasachPlayYT** [comment]
У меня такая проблема возникаем тоже вообще с любыми методами обхода ТОЛЬКО на пк и в оффициальном плеере. Еще я узнал, что часть куки подсасывается в режиме инкогнито, поэтому дело может быть в них. На телефоне и телевизорах стоит byeDPI 1.2.0, DNS 1.1.1.1 и настройки -n “www.google.com” -q 1+s -O 1 -s 25+s -t 5, видосы иногда могут не подгружаться, я хз как это исправить, но в целом можно смотреть.
